crates/lovely-win/Cargo.toml

```diff
@@ -14,6 +14,7 @@ itertools = "0.13.0"
 libc = "0.2.141"
 once_cell = "1.19.0"
 widestring = "1.0.2"
+log = "0.4.21"
```

Comment:
Ditto from above, this doesn't seem to be necessary? I could be missing something because it's absolutely late.